packages/resource-deployment/scripts/pool-startup/pool-startup-internal.sh

```diff
@@ -15,6 +15,8 @@ waitForAptUpdates() {
     done
 }
 
+waitForAptUpdates
```

Comment:
the very first time when the node starts, dpkg configuration fails sometimes due to lock issues.